### DISCUSSION TITLE
[BUG] MGTK disguise no image (only allies)

### DISCUSSION THREAD

**Discussion by Ra2Liker** [Phobos-developers/Phobos#1260] [Ideas and feedback]
### Description

[BUG] MGTK disguise no image (only allies)
![Image_1715776749872](https://github.com/Phobos-developers/Phobos/assets/169903303/a1d808cb-2a03-4e12-b78f-d5e3bce257df)

### Checklist

- [X] The suggestion is **NOT** related to functionality of Ares, HAres, Kratos or any other YR engine extension.
- [X] The suggested feature / enhancement is **NOT** implemented in Ares/Phobos yet and the existing possibilites are not viable or good enough to achieve the wanted result.
- [X] I agree to elaborate the details if requested and provide thorough testing if the feature is implemented.
- [X] I added a very descriptive title to this topic.
- [X] I used the GitHub search and read the topic list to find a similar topic and didn't find it.
- [X] I thought out how the feature / enhancement should work and described it in the most detailed and clear way.

**Comment by Metadorius** [comment]
What is the problem even? How is this related to Phobos?
Also why it is not in issues if it's really a problem?

**Comment by Ra2Liker** [comment]
In the PhobosB40 have "CanDisguise=yes" Vehicle (like a Mirage Tank)


When&nbsp;Belong to allies does not have any images
（only Ares no this BUG）










------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2024年5月15日(星期三) 晚上8:57
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [Phobos-developers/Phobos] [BUG] MGTK disguise no image (only allies) (Discussion #1260)














 
What is the problem even? How is this related to Phobos?

**Comment by FS-21** [comment]
Looks a duplicate of issue #1244

**Reply by Ra2Liker** [reply]
ok...It was my mistake
